package/MDAnalysis/__init__.py

```diff
@@ -200,3 +200,37 @@
 from .coordinates.MMTF import fetch_mmtf
 
 from .migration.ten2eleven import ten2eleven
+
+from .due import due, Doi, BibTeX
+
+due.cite(BibTeX((
+            "@inproceedings{gowers2016, "
+            "title={MDAnalysis: A Python package for the rapid analysis "
+            "of molecular dynamics simulations}, " 
+            "author={R. J. Gowers and M. Linke and "
+            "J. Barnoud and T. J. E. Reddy and M. N. Melo "
+            "and S. L. Seyler and D. L. Dotson and J. Domanski and "
+            "S. Buchoux and I. M. Kenney and O. Beckstein},"
+            "journal={Proceedings of the 15th Python in Science Conference}, "
+            "pages={102-109}, "
+            "year={2016}, "
+            "editor={S. Benthall and S. Rostrup}, "
+            "note={Austin, TX, SciPy.} "
+            "}"
+            )), 
+            description=
+         "Description of recent updates and new functionality "
+         "in MDAnalysis since release 0.7.2 (March 2011) up to "
+         "release 0.16.0 (April 2017). Please cite together with "
+         "Michaud-Agrawal et al 2011 ",
+         path="MDAnalysis/", cite_module=True)
+due.cite(Doi("10.1002/jcc.21787"),
+         description=
+         "Description of MDAnalysis up to release 0.7.2 (March 2011), "
+         "including the basic philosophy of the library and the "
+         "LeafletFinder algorithm. Please cite together with "
+         "Gowers et al 2016, "
+         "J. Comput. Chem. 32 (2011), 2319-2327 ",
```

Comment:
Just to clarify -- are you providing less info (i.e., all author names, etc.) for the original MDA paper just because you prefer citation of the newer proceedings paper?

Reply:
I would prefer if we give full information for all papers.

Reply:
No, the full citation information is obtained via `DOI()`. The current output is
```
In [1]: import MDAnalysis

In [2]:
Do you really want to exit ([y]/n)?

DueCredit Report:
- Description of MDAnalysis up to release 0.7.2 (March 2011), including the basic philosophy of the library and the LeafletFinder algorithm. Please cite together with Gowers et al 2016, J. Comput. Chem. 32 (2011), 2319-2327 , Description of recent updates and new functionality in MDAnalysis since release 0.7.2 (March 2011) up to release 0.16.0 (April 2017). Please cite together with Michaud-Agrawal et al 2011  / MDAnalysis/ (v None) [1, 2]
- Scientific tools library / numpy (v 1.14.1) [3]

2 packages cited
0 modules cited
0 functions cited

References
----------

[1] Gowers, R.J. et al., 2016. MDAnalysis: A Python package for the rapid analysis of molecular dynamics simulations. In S. Benthall & S. Rostrup, eds. Proceedings of the 15th Python in Science Conference. pp. 102–109.
[2] Michaud-Agrawal, N. et al., 2011. MDAnalysis: A toolkit for the analysis of molecular dynamics simulations. Journal of Computational Chemistry, 32(10), pp.2319–2327.
[3] Van Der Walt, S., Colbert, S.C. & Varoquaux, G., 2011. The NumPy array: a structure for efficient numerical computation. Computing in Science & Engineering, 13(2), pp.22–30.
```

Reply:
I understand your confusion: the string `"J. Comput. Chem. 32 (2011), 2319-2327 "` needed to be removed...